src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1801,12 +1804,15 @@ public void addError(AuditEvent event) {
             final IndentComment comment = comments[position];
             position++;
 
+            final String possibleExceptedMessages = Arrays.stream(comment.getExpectedMessages())
+                    .reduce("", (cur, next) -> cur + "\"" + next + "\", ");
+            final boolean hasExpectedMessagePostfix =
+                    Arrays.stream(comment.getExpectedMessages()).anyMatch(message::endsWith);
```

Comment:
what do you think about removing this variable and inlining condition to assertion below?